style.css

```diff
@@ -0,0 +1,86 @@
+h1{	
+		font-size: 80px;
+		text-align: center; 
+		color: Yellow;
+		margin-bottom: -10px;
+		text-decoration: underline;
+
+	}
+
+h3{
+		text-align: center; 
+		color: blue;
+	}
+
+h3.flea {
+		text-align:left;
+}	
+
+p {
+		margin-left: 30px;
+		margin-right: 20px;
+		line-height: 25px;
+		margin-bottom: 50px;
+	}
+
+body {
+
+		font-family: sans-serif ;
+
+		background-color:#D2D2D2;
+		color:black;
+		margin-left: 30px;
+		margin-bottom: 10px;
+		margin-right: 30px;
+	}
+
+img.action  {	
+			border: solid 5px;
```

Comment:
@LuisLithgow You didn't declare a color here, which works, but just be aware that if you wanted a specific color (other than black) you'd need to state it like so:
`border: 5px solid blue`